code/modules/mob/living/simple_animal/abnormality/teth/drowned_sisters.dm

```diff
@@ -25,15 +26,69 @@
 	gift_type =  /datum/ego_gifts/sorority
 	abnormality_origin = ABNORMALITY_ORIGIN_WONDERLAB
 
+	var/breaching = FALSE
 
-/mob/living/simple_animal/hostile/abnormality/drownedsisters/AttemptWork(mob/living/carbon/human/user, work_type)
-	//Deals scaling work damage based off your stats.
+//Work Mechanics
+/mob/living/simple_animal/hostile/abnormality/drownedsisters/AttemptWork(mob/living/carbon/human/user, work_type)	//Deals scaling work damage based off your stats.
+	if(breaching)
+		return FALSE
 	work_damage_amount = (get_attribute_level(user, PRUDENCE_ATTRIBUTE) -60) * -0.5
-	work_damage_amount = max(1, work_damage_amount)	//So you don't get healing
-	return TRUE
+	work_damage_amount = max(5, work_damage_amount)	//So you don't get healing
+	return ..()
 
 /mob/living/simple_animal/hostile/abnormality/drownedsisters/PostWorkEffect(mob/living/carbon/human/user, work_type, pe, work_time)
 // okay so according to the lore you're not really supposed to remember the stories she says so we're going to make it so your sanity goes back up
 	if(!user.sanity_lost && pe != 0)
 		user.adjustSanityLoss(-get_attribute_level(user, PRUDENCE_ATTRIBUTE))
 	..()
+	switch(work_type)
+		if(ABNORMALITY_WORK_INSTINCT)
+			datum_reference.qliphoth_change(-1)
+		if(ABNORMALITY_WORK_ATTACHMENT)
+			if(datum_reference.qliphoth_meter == 3)
+				FloodRoom()
+			else
+				datum_reference.qliphoth_change(1)
+	return
+
+//Breaches
+/mob/living/simple_animal/hostile/abnormality/drownedsisters/ZeroQliphoth(mob/living/carbon/human/user)
+	datum_reference.qliphoth_change(3)
+	if(!user)
+		return
+	to_chat(user, "<span class='userdanger'>You are attacked by an invisible assailant!</span>")
+	playsound(get_turf(src), 'sound/abnormalities/jangsan/tigerbite.ogg', 75, 0)
+	user.apply_damage(200, RED_DAMAGE, null, user.run_armor_check(null, BLACK_DAMAGE))
+	if(user.health < 0 || user.stat == DEAD)
+		user.gib()
+	return
+
+/mob/living/simple_animal/hostile/abnormality/drownedsisters/proc/FloodRoom() //Qliphoth Went over max
+	breaching = TRUE
+	for(var/turf/open/T in view(7, src))
+		new /obj/effect/temp_visual/water_waves(T)
+	playsound(get_turf(src), 'sound/abnormalities/piscinemermaid/waterjump.ogg', 75, 0)
+	var/list/teleport_potential = list()
+	for(var/turf/T in GLOB.department_centers)
+		teleport_potential += T
+	if(!LAZYLEN(teleport_potential))
+		return
+	for(var/mob/living/carbon/human/H in view(7, src))
+		if(!H.sanity_lost)
+			var/turf/teleport_target = pick(teleport_potential)
+			TeleportPerson(H, teleport_target)
+		else
+			QDEL_NULL(H.ai_controller) //If they panicked, they just drown
+			H.adjustOxyLoss(200)
+	sleep(5 SECONDS)
+	datum_reference.qliphoth_meter = 3
```

Comment:
```suggestion
	datum_reference.qliphoth_change(3)
```
Method is calls a lot of important things, including logging and visual/audio alerts.